src/python/WMCore/Services/TagCollector/TagCollector.py

```diff
@@ -112,3 +112,38 @@ def releases_by_architecture(self):
                 releases.add(item['label'])
             arch_dict[row['name']].extend(list(releases))
         return dict(arch_dict)
+
+
+    def defaultMicroArchVersionNumberByRelease(self, default_microarch=2):
```

Comment:
Kenyi, please create a unit test for this new method and also update the docstring according to the contribution guidelines: https://github.com/dmwm/WMCore/blob/master/CONTRIBUTING.rst#project-docstrings-best-practices